libs/clients/national-registry/v1/src/lib/nationalRegistryApi.ts

```diff
@@ -123,6 +123,33 @@ export class NationalRegistryApi {
     return Array.isArray(documentData) ? documentData : [documentData]
   }
 
+  public async postUserCorrection(
+    values: FamilyCorrection,
+  ): Promise<{ success: boolean; message?: string }> {
+    const response = await this.signal(
+      'CreateAndUpdateMS_Leidretting',
+      {
+        S5RequestID: '',
+        Kennitala: values.nationalId,
+        Barn: values.nationalIdChild,
+        Nafn: values.name,
+        Simanumer: values.phonenumber,
+        Netfang: values.email,
+        Athugasemd: values.comment,
+        Stada: 'Skráð',
+      },
+      true,
+    )
+
+    if (!response) {
+      throw new ForbiddenException(`User correction not sent. Unknown error`)
```

Comment:
Throw a different type of error, since this isn't really forbidden exception, and we most always take different actions based on forbidden exceptions in the apollo's error link.

Reply:
yes, makes sense 💯